apps/frontend/src/libs/components/auth-wrapper/styles.module.css

```
@@ -1,10 +1,23 @@
 .page-layout {
 	display: flex;
-	height: 100vh;
+	height: 100%;
```

```suggestion
	min-height: 100vh;
```
Wouldn't it be better?

---

We already have min-height for `body` so I think % it is fine there